File: SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/Comentario/ComentarioServicioDTO.java
package pe.edu.pucp.softinv.model.Comentario;

import pe.edu.pucp.softinv.model.Personas.ClienteDTO;
import pe.edu.pucp.softinv.model.Servicio.ServicioDTO;

public class ComentarioServicioDTO extends ComentarioDTO {
    private ServicioDTO servicio;

    public ServicioDTO getServicio() {
        return servicio;
    }

    public void setServicio(ServicioDTO servicio) {
        this.servicio = servicio;
    }

    public ComentarioServicioDTO() {
        super();
        servicio = null;
    }

    public ComentarioServicioDTO(Integer idComentario, ClienteDTO cliente, String comentario, Integer valoracion, ServicioDTO servicio) {
        super(idComentario,cliente,comentario,valoracion);
        this.servicio = servicio;
    }

    @Override
    public Integer getIdProducto(){
        return null;
    }

    @Override
    public Integer getIdServicio(){
        return servicio.getIdServicio();
    }

}

File: C#/SoftInvModel/Comentario/comentarioServicioDTO.cs
using System;

namespace softinv.model
{
    public class ComentarioServicioDTO: ComentarioDTO
    {
        private ServicioDTO servicio;

        public ServicioDTO Servicio { get => servicio; set => servicio = value; }

        public ComentarioServicioDTO():base()
        {
            Servicio = null;
        }

        public ComentarioServicioDTO(int idComentario, ClienteDTO cliente, String comentario, int valoracion, ServicioDTO servicio): base(idComentario, cliente, comentario, valoracion)
        {
            this.Servicio = servicio;
        }

        public int? IdServicio { get => servicio.IdServicio; set => servicio.IdServicio = value; }

    }

}
